Jantimizer/QueryParser/QueryParsers/PostgresParser.cs

```diff
@@ -37,19 +37,24 @@ public async Task<ParserResult> ParseQueryAsync(string query)
         {
             string explanationTextBlock = await GetPGExplainationTextBlock(query);
 
+            return AnalyseExplanationText(explanationTextBlock);
+        }
+
+        public static ParserResult AnalyseExplanationText(string explanationText)

```

Comment:
Lots of static methods here? why? its an instantiated class anyway, so you would have to make a `new PostgresParser(...)` anyway, before you could even call this function.
Same goes for the rest of the static functions in this class.